mavis/Extractor.h

```diff
@@ -306,12 +306,20 @@ class ExtractorBase : public ExtractorIF
 
     void print(std::ostream& os) const override
     {
-        os << "Extractor '" << getName() << "'"
-           << std::endl;
+        os << "Extractor '" << getName() << "'";
```

Comment:
The removal of the `std::endl` here accounts for a lot of the changes to `golden.out`